tests/core/test_settings.py

```diff
@@ -0,0 +1,26 @@
+from tests.module_helper import PyttmanInternalBaseTestCase
+from pyttman.core.internals import Settings
+
+from importlib import import_module
+from . import mocksettings
```

Comment:
I think this file is redundant, and the relative import can be troublesome in a shipped package due to its relativity. Can the mockup dict object be placed in the test suite instead, such as in `setUp`, or as a property in the test suite?